mathics/doc/doc_entries.py

```diff
@@ -406,9 +462,11 @@ def get_tests(self) -> list:
         return []
 
     def is_private(self) -> bool:
+        """the test is private"""
```

Comment:
See above. (The value of a docstring here would be to explain what "private" means.